### ISSUE TITLE
Rework method useQuery hooks to enabled only when signclients/clients/signers ready

### ISSUE THREAD

**Issue by codingki** [graz-sh/graz#91]
Example: 
from 
```tsx
export const useBalances = (bech32Address?: string): UseQueryResult<Coin[]> => {
  const { data: account } = useAccount();
  const address = bech32Address || account?.bech32Address;

  const queryKey = ["USE_BALANCES", address] as const;
  const query = useQuery(queryKey, ({ queryKey: [, _address] }) => getBalances(_address!), {
    enabled: Boolean(address),
    refetchOnMount: false,
    refetchOnReconnect: true,
    refetchOnWindowFocus: false,
  });

  return query;
};
```

to 
```tsx
export const useBalances = (bech32Address?: string): UseQueryResult<Coin[]> => {
const { signingClients } = useGrazSessionStore.getState();
  const { data: account } = useAccount();
  const address = bech32Address || account?.bech32Address;

  const queryKey = ["USE_BALANCES", address] as const;
  const query = useQuery(queryKey, ({ queryKey: [, _address] }) => getBalances(_address!), {
    enabled: Boolean(address) || Boolean(signingClients),
    refetchOnMount: false,
    refetchOnReconnect: true,
    refetchOnWindowFocus: false,
  });

  return query;
};
```

**Comment by codingki** [comment]
will be added in `0.1.x`

**Comment by grikomsn** [comment]
Closing this as already covered by the current hook behavior.

Findings:
- The original ask was to keep method/query hooks disabled until their clients, signing clients, or signers are ready.
- `useBalances`, `useBalance`, and `useBalanceStaked` now gate on required args and only enable their final queries once a `StargateClient` and chain are available.
- `useStargateSigningClient` and `useCosmWasmSigningClient` gate on configured chains, wallet presence, the user-provided `enabled` flag, and connected session state; inactive chains return `null` instead of trying to create a signing client.
- `useOfflineSigners` is also gated on connected wallet/session state.
- Focused coverage exists for this readiness behavior in `src/hooks/__tests__/account-balances.test.tsx` and `src/hooks/__tests__/clients-signing.test.tsx`. I re-ran those tests with `pnpm --dir packages/graz test --run src/hooks/__tests__/account-balances.test.tsx src/hooks/__tests__/clients-signing.test.tsx`: 2 files passed, 8 tests passed.

There may still be room for a tiny cleanup to make the readiness booleans subscribe to Zustand state more idiomatically instead of reading some pieces through `getState()`, but that is a separate robustness/refactor task rather than the feature requested here.
